node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -211,6 +222,7 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
             vertices_to_be_added = self.vertices_to_be_added_len(),
             "synchronizer queue lengths"
         );
+        debug!(oldest_panorama=%self.oldest_seen_panorama, "oldest seen unit per validator");
```

Comment:
We could probably stop logging this once all entries have sequence number 0.

Reply:
I was also looking into how it would update if we started purging the vertices from the synchronizer but I am not sure if it's worth it – we would have to iterate through all entries in the `self.vertices_to_be_added` collection.

Reply:
Done in [`0a9a01f` (#1173)](https://github.com/CasperLabs/casper-node/pull/1173/commits/0a9a01fb6570e0dbd21236720e8faa2c30a529c8).